findbugs-exclude.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<FindBugsFilter>
+    <Match>
+        <Bug pattern="URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD,BC_UNCONFIRMED_CAST,BC_UNCONFIRMED_CAST_OF_RETURN_VALUE"/>
+    </Match>
+    <Match>
+        <Package name="org.ballerinalang.docgen.model"/>
+        <Package name="org.ballerinalang.docgen.Generator"/>
```

Comment:
This should be Class instead of Package.

Reply:
Fixed with 93bb371